### PR TITLE
Cors true

### PR DESCRIPTION
**Pull request by PhilNorman2**
Access to fetch at 'http://localhost:8888/api/agencies?zip_code=43008&event_date=2020-04-25' from origin 'http://localhost:3000' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
App.js:26 GET http://localhost:8888/api/agencies?zip_code=43008&event_date=2020-04-25 net::ERR_FAILED

To avoid this problem.  Insert the line “config.cors = true”  in your config/appliation.rb file

Jets.application.configure do
  config.project_name = 'pantry-finder-api'
  config.mode = 'api'
  config.cors = true

This will will add a response header with Access-Control-Allow-Origin=‘*’
See https://rubyonjets.com/docs/cors-support/ for more info.

### CODE REVIEW

File: pantry-finder-api/config/application.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Jets.application.configure do
   config.project_name = 'pantry-finder-api'
   config.mode = 'api'
+  config.cors = true
```
Comment (nvengal) [inline pantry-finder-api/config/application.rb:6]: This is fine but it looks like changes from your other pr got muddled into here. Not a big deal but might lead to some merge conflicts.

**Comment by PhilNorman2** [comment]
The pull requests are pulling in more commits than I would like.  PR #45 is pulling in the same changes as #44.  Perhaps we just approve them one at a time.  One snafu, I deleted the foodbank files in this PR thinking that might remove them from the pull request but instead they will be removed from the repository.  Another alternative might be to close the PR but I am afraid that might leave the commit around to get picked up in subsequent pull requests.

**Comment by nvengal** [comment]
I think what's happening is that when you go to make a branch for a pr instead of going from `master -> new branch` you're starting a pr off of an existing pr `existing pr branch -> new branch`. I can fix this one up for now. Going to push some changes to your branch.

**Comment by nvengal** [comment]
This issue seems to be effecting us as well https://github.com/codecov/codecov-action/issues/69

**Pull request merged**